app/views/sitemap/index.xml.haml

```diff
@@ -0,0 +1,17 @@
+!!! XML
+%urlset{xmlns: "http://www.sitemaps.org/schemas/sitemap/0.9"}
+  - for page in @pages
+    %url
+      %loc= root_url + page
+      %changefreq monthly
+
+  - for enterprise in @enterprises
+    %url
+      %loc= enterprise_shop_url(enterprise)
+      %lastmod= enterprise.updated_at.strftime('%Y-%m-%d')
+      %changefreq weekly
+
+  - for group in @groups
+    %url
+      %loc= group_url(group)
+      %changefreq yearly
```

Comment:
This seems like a long time for groups, especially as we don't have data on how often they change. What do you reckon of using monthly here instead?